ravenframework/SupervisedLearning/FeatureSelection/utils.py

```diff
@@ -64,8 +64,8 @@ def screenAndTrainEstimator(Xreduced, yreduced, estimator, support, params, incl
     @ In, addOnKeys, list, optional, list of additional keys to remove
     @ Out, None
   """
-  assert (not issubclass(estimator.__class__, SupervisedLearning.SupervisedLearning),
-          f"estimator class str(estimator.__class__) is not a SupervisedLearning derived class")
+  assert not issubclass(estimator.__class__, SupervisedLearning.SupervisedLearning), \
```

Comment:
I think you should remove "not" from this check. I have checked on my computer, and tests passed. @dylanjm